### PR TITLE
Pan-OS - full logs UC

### PR DESCRIPTION
**Pull request by bakatzir**
@David-BMS FYI

## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/18217

## Description
descriptions of outputs are according to: https://docs.paloaltonetworks.com/pan-os/8-0/pan-os-admin/monitoring/use-syslog-for-monitoring/syslog-field-descriptions/threat-log-fields

Also deprecated the old traffic log commands.

## Required version of Demisto
3.0.0

## Does it break backward compatibility?
   - No

## Must have
- [x] Tests - yes - both on unit tests, and on new PB using the newly implemented commands
- [x] Documentation - https://github.com/demisto/etc/issues/18658
- [x] Code Review

### CODE REVIEW

File: Integrations/Panorama/Panorama.py [hunk 1]
```diff
@@ -3272,6 +3278,331 @@ def panorama_get_traffic_logs_command():
...
+    query_logs_output = {
+        'JobID': job_id,
+        'Status': 'Pending'
+    }
+
+    if 'response' not in result or 'result' not in result['response'] or 'job' not in result['response']['result']\
+            or 'status' not in result['response']['result']['job']:
+        return_error('Missing JobID status in response')
+
+    if result['response']['result']['job']['status'] != 'FIN':
```
Comment (bakatzir) [inline Integrations/Panorama/Panorama.py]: @yuvalbenshalom If the logs aren't done, i'm updating the context with the JobID and the Pending status.

**Comment by bakatzir** [comment]
@kirbles19 please see that deprecated commands have a change in the command name in the UI. So please take that into account while deciding on the deprecated commands description.
the deprecated are:
`panorama-query-traffic-logs`
`panorama-check-traffic-logs-status`
`panorama-get-traffic-logs`
Screen Shot 2019-08-14 at 12 49 32

**Comment by bakatzir** [comment]
Integration:
https://github.com/demisto/content/blob/5b6d0eb47d6edc703f07714766bc12dc5af999fe/Integrations/Panorama/CHANGELOG.md
https://github.com/demisto/content/blob/5b6d0eb47d6edc703f07714766bc12dc5af999fe/Integrations/Panorama/Panorama.yml
Playbook:
https://github.com/demisto/content/blob/5b6d0eb47d6edc703f07714766bc12dc5af999fe/Playbooks/playbook-Panorama_Query_Logs.yml
https://github.com/demisto/content/blob/5b6d0eb47d6edc703f07714766bc12dc5af999fe/Playbooks/playbook-Panorama_Query_Logs_CHANGELOG.md

**Comment by kirbles19** [comment]
@bakatzir @yuvalbenshalom all files reviewed, all done.

**Pull request merged**